src/commands/promote.js

```diff
@@ -0,0 +1,73 @@
+const Command = require('../base');
+const { flags: flagUtils } = require('@oclif/command');
+
+class PromoteCommand extends Command {
+  /**
+   * Runs the promote command for a specific version of a package that has already been built
+   *
+   * @async
+   * @returns {Promise} The result or error
+   */
+  async run() {
+    const { flags, args } = this.parse(PromoteCommand);
+    const { env } = args; // package is reserved so we don't try to destructure it
+    const { pkg, version } = this.parsePackage(args.package);
+    const { wrhsHost, auth } = this.mergeConfig(flags);
+
+    if (!wrhsHost) {
+      this.error(this.missingHostError());
+    }
+    if (!version) {
+      this.error(this.missingVersionError());
```

Comment:
I feel like these error cases should be `return`ed? In order to prevent continuing the function execution?

Reply:
> The tests failing are very strange. It seems like it should pass. does this reproduce locally @wjarvis1?
thanks for pointing that out! Actually didn't notice this earlier. Yes, it is reproducing locally. It is strange cuz all tests have green checkmark next to them. Looking into the issue rn

Reply:
Just checked the `oclif` docs, and calling `error` kills the process, so the return isn't really needed. Might make the code clearer though.

https://oclif.io/docs/commands#thiserrormessage-string-error-options-code-string-exit-number